e2e/cypress/integration/interactive_dialog/boolean_value_check_spec.js

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @interactive_dialog
+
+describe('Messaging', () => {
+    let testChannel;
+
+    before(() => {
+        // # Login as test user and visit the newly created test channel
+        cy.apiInitSetup().then(({team, user, channel}) => {
+            testChannel = channel;
+
+            // # Set up Demo plugin
+            cy.apiInstallPluginFromUrl('https://github.com/mattermost/mattermost-plugin-demo/releases/download/v0.8.0/com.mattermost.demo-plugin-0.8.0.tar.gz', true);
+            cy.apiEnablePluginById('com.mattermost.demo-plugin');
+
+            // # Login as regular user
+            cy.apiLogin(user);
+            cy.visit(`/${team.name}/channels/${testChannel.name}`);
```

Comment:
Add an additional step to wait till postbox is visible. 
`cy.get('#post_textbox', {timeout: TIMEOUTS.ONE_MIN}).should('be.visible');`